docs/querying/api.md

```diff
@@ -382,6 +382,79 @@ $ curl http://localhost:9090/api/v1/alertmanagers
 }
 ```
 
+## Status
+
+Following status endpoints expose current Prometheus configuration.
+
+### Config
+
+The following endpoint returns currently loaded configuration file:
+
+```
+GET /api/v1/status/config
+```
+
+Currently, config is returned as dumped YAML file.
```

Comment:
The "currently" doesn't add anything here, we won't be changing that as it'd be a breaking change. You might mention though that comments are not present, due to limitations of the yaml library we use.